src/test/java/uk/gov/hmcts/reform/profilesync/util/UserProfileSyncJobSchedulerTest.java

```diff
@@ -2,81 +2,84 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
move it schedular package

Reply:
this has been addressed in the following commit: https://github.com/hmcts/rd-profile-sync/pull/181/commits/bc37603637e5fcbbc46bdd8621f47415d846e2c2